src/main/java/com/minecolonies/coremod/compatibility/jei/FishermanRecipeCategory.java

```diff
@@ -0,0 +1,137 @@
+package com.minecolonies.coremod.compatibility.jei;
+
+import com.minecolonies.api.colony.buildings.ModBuildings;
+import com.minecolonies.api.colony.jobs.ModJobs;
+import com.minecolonies.api.loot.ModLootTables;
+import com.minecolonies.api.util.constant.TranslationConstants;
+import com.minecolonies.coremod.colony.crafting.CustomRecipeManager;
+import com.minecolonies.coremod.colony.crafting.LootTableAnalyzer;
+import mezz.jei.api.constants.VanillaTypes;
+import mezz.jei.api.gui.IRecipeLayout;
+import mezz.jei.api.gui.ingredient.IGuiItemStackGroup;
+import mezz.jei.api.helpers.IGuiHelper;
+import mezz.jei.api.ingredients.IIngredients;
+import net.minecraft.item.ItemStack;
+import net.minecraft.loot.LootTables;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.util.text.TranslationTextComponent;
+import net.minecraftforge.api.distmarker.Dist;
+import net.minecraftforge.api.distmarker.OnlyIn;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+/**
+ * The JEI recipe category for the fisherman.
+ */
+@OnlyIn(Dist.CLIENT)
+public class FishermanRecipeCategory extends JobBasedRecipeCategory<FishermanRecipeCategory.FishingRecipe>
+{
+    public FishermanRecipeCategory(@NotNull final IGuiHelper guiHelper)
+    {
+        super(ModJobs.fisherman.getHandlerProducer().apply(null), ModJobs.FISHERMAN_ID,
+                new ItemStack(ModBuildings.fisherman.getBuildingBlock()), guiHelper);
+    }
+
+    private static final int LOOT_SLOTS_X = CITIZEN_X + CITIZEN_W + 4;
+    private static final int LOOT_SLOTS_W = WIDTH - LOOT_SLOTS_X;
+
+    @NotNull
+    @Override
+    public Class<? extends FishingRecipe> getRecipeClass() {

```

Comment:
minor formatting thing

Reply:
Fixed.  Dangers of auto-collapsing single-line methods.